demo/src/test/java/com/workday/autoparse/json/demo/InstanceUpdaterTest.java

```diff
@@ -353,6 +353,18 @@ public void testJSONArray() throws Exception {
         assertTrue("myJsonArray[2] instanceof JSONObject", testObject.myJsonArray.opt(2) instanceof JSONObject);
     }
 
+    @Test
+    public void testCollectionWithNullValueUpdate() throws Exception {
+        TestObject testObject = new TestObject();
+        testObject.myCollectionWithNullValues = Arrays.asList("test", null, "null");;
```

Comment:
`CollectionUtils.newArrayList()`. And you have one too many semicolons.

Reply:
Done.